src/components/AssetSearch/hooks/useGetPopularAssetsQuery.tsx

```diff
@@ -0,0 +1,39 @@
+import { type ChainId } from '@shapeshiftoss/caip'
+import type { Asset } from '@shapeshiftoss/types'
+import { useQuery } from '@tanstack/react-query'
+import { getMarketServiceManager } from 'state/slices/marketDataSlice/marketServiceManagerSingleton'
+import { selectAssets } from 'state/slices/selectors'
+import { store } from 'state/store'
+
+const queryKey = ['getPopularAssetsQuery']
+
+export const queryFn = async () => {
+  const assetIds = await getMarketServiceManager().getPopularAssets(100)
```

Comment:
Note, market-service-manager itself will cache things for the lifecycle of the app, so refetchy bits may not work as expected if we ever want to add them.

Looking at `staleTime: Infinity` looks like the intent is to get this stable forever for the tab lifecycle so we should be good, but just nothing this here in case we would expect to make things refetchy in the future!